cockatrice/src/carditem.cpp

```diff
@@ -16,6 +17,7 @@
 #include "settingscache.h"
 #include "tab_game.h"
 #include "pb/serverinfo_card.pb.h"
+#include "abstractcarditem.h"
```

Comment:
CardItem **is** an AbstractCardItem, so will inherit its parents methods.

You can remove both imports and just have the `!getFaceDown()`